data/import_data.R

```diff
@@ -1,8 +1,9 @@
 # data from tidytuesday
 # https://github.com/rfordatascience/tidytuesday/blob/master/data/2020/2020-03-31/readme.md
```

Comment:
Does it make sense to read in the CSVs from the tidytuesday repo instead of/in-addition-to storing them in this repository? (from a data-sourcing perspective)

Reply:
i.e. 
```
brewing_materials <- readr::read_csv('https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2020/2020-03-31/brewing_materials.csv')
beer_taxed <- readr::read_csv('https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2020/2020-03-31/beer_taxed.csv')
brewer_size <- readr::read_csv('https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2020/2020-03-31/brewer_size.csv')
beer_states <- readr::read_csv('https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2020/2020-03-31/beer_states.csv')

Reply:
Sure I was wondering what people would prefer, I'll just include that in the .R file and remove the csvs :)

Reply:
Replaced with script above